src/test/java/org/casbin/jcasbin/main/EnforcerUnitTest.java

```diff
@@ -39,7 +39,7 @@ public void testKeyMatchModelInMemory() {
         m.addDef("r", "r", "sub, obj, act");
         m.addDef("p", "p", "sub, obj, act");
         m.addDef("e", "e", "some(where (p.eft == allow))");
-        m.addDef("m", "m", "r.sub == p.sub && keyMatch(r.obj, p.obj) && regexMatch(r.act, p.act)");
+        m.addDef("m", "m", "r.sub == p.sub && keyMatch(r.obj, p.obj) && regexMatch(r.act, p.act) && globMatch(r.obj, p.obj)");
```

Comment:
Please copy the tests from Go: https://github.com/casbin/casbin/pull/397/files

1. glob_model.conf and glob_policy.csv
2. unit tests like `testGlobMatch(t, "/foo", "/foo", true)`